apps/public-docsite-resources/config/pre-copy.json

```diff
@@ -1,10 +1,6 @@
 {
   "copyTo": {
```

Comment:
@ecraig12345 The charting demo app generated for this PR is still not loading.
https://fluentuipr.z22.web.core.windows.net/pull/22054/react-charting/demo/index.html

Please take a look.

Reply:
Turns out the pages failing to load was a different issue, #22076